.github/workflows/python-package.yml

```diff
@@ -41,8 +41,14 @@ jobs:
         pip install -U setuptools>=40.1 wheel 'cmake<3.20'
         pip install $(pwd)/scanpy-scripts
         python -m pip install $(pwd)/scanpy --no-deps --ignore-installed -vv
+        pip install pytest
```

Comment:
Just to be tidy, could you put these deps in the install statement on line 41?

Reply:
Dine in 378b115cbf1f64b1f4d5350469c95e77f0fec8fa